public/js/roster.js

```diff
@@ -60,9 +60,13 @@ function createTable() {
                                 ${ row.name }
                             </a>
                         </li>
-
-                        ${ row.raid_name || row.class ? `
+                        ${ row.is_alt || row.raid_name || row.class ? `
                             <li>
+                                ${row.is_alt ? `
+                                <span class="font-weight-bold">
+                                    <span class="role-circle" style="color:orange">Alt</span>
+                                </span>
+                                ` : ''}

```

Comment:
Please change to:
```

${ row.is_alt ? `
        <span class="text-legendary font-weight-bold">Alt</span>&nbsp;
` : '' }
```

![image](https://user-images.githubusercontent.com/3407841/90582109-cfb50800-e181-11ea-96b3-ef2cc49ecd0f.png)

![image](https://user-images.githubusercontent.com/3407841/90582119-d5125280-e181-11ea-8f7e-2f5c2f1241e3.png)